integration_tests/test_cargo_audit.py

```diff
@@ -0,0 +1,10 @@
+from subprocess import Popen, PIPE, STDOUT
+
+
+def test_cargo_audit():
+    cmd = 'cargo-audit audit'
+    prs = Popen("{}".format(cmd), shell=True, stdin=PIPE,
+            stdout=PIPE, stderr=STDOUT, close_fds=True)
+    stdout, nothing = prs.communicate()
```

Comment:
Can we use `subprocess.run` instead? You can take a look at an example here: https://github.com/rust-vmm/rust-vmm-ci/blob/9dd386c2e6d35aabdb2064f3ca0d17f2ffd26d4b/integration_tests/test_commit_format.py#L26

We need to add `check = True` otherwise when there are vulnerable dependencies the test pass even though it shouldn't.